client/agent-opentracing/src/main/resources/apmrules/javax/apm-javax-jms.btm

```diff
@@ -0,0 +1,47 @@
+# Does not include monitoring for temporary queues/topics currently
+
+RULE javax.jms(1) Javax JMS Consumer Async Start
+INTERFACE ^javax.jms.MessageListener
+METHOD onMessage(javax.jms.Message)
+HELPER org.hawkular.apm.agent.opentracing.OpenTracingManager
+AT ENTRY
+BIND spanCtx:SpanContext = getTracer().extract(textMapFormat(),
```

Comment:
Again unfortunately not, byteman does not support it :( so this is the workaround.